README.md

```diff
@@ -1,2 +1,25 @@
 # pyrsh
 Remote shell in python
+
+to run in cmd you need to enter
+`pyrsh.py type command user password host port`
+
+type - connection type
+    pyr_local - running cmd on the local machine
```

Comment:
@GitSergeyGit can we reduce `pyr_` prefix for all the types? For the `paramiko` we can use `ssh_paramiko`